utilities/tags/addTagsUsingCSV.py

```diff
@@ -0,0 +1,44 @@
+import csv
+import requests
+
+SEMGREP_API_KEY = ""
+API_URL = "https://semgrep.dev/api/v1/deployments/{deployment-slug}/projects/{project_name}/tags"
+# Path to your CSV file
+CSV_FILE = "projects_tags.csv"
```

Comment:
Ideally we'd have this come in as a CLI option too, but I wouldn't block on that.